modules/AWS/rds/main.tf

```diff
@@ -23,7 +23,7 @@ module "security_group" {
 
 module "db" {
   source  = "terraform-aws-modules/rds/aws"
-  version = "~> 3.0"
+  version = "~> 3.5"
```

Comment:
Any reason for the minor bump here - assuming its latest?

Reply:
With the pessimistic operator I assume we're fine if versions `3.5` -> `3.9` can be used?

Reply:
The latest is 4.1.2, we had to upgrade the RDS version as there is a deprecated input causing a warning message when we run terraform:
```
Warning: Argument is deprecated

  with module.bamboo.module.database.module.db.module.db_instance.aws_db_instance.this,
  on .terraform/modules/bamboo.database.db/modules/db_instance/main.tf line 34, in resource "aws_db_instance" "this":
  34:   name                                = var.name

Use db_name instead

(and 9 more similar warnings elsewhere)
```

I just double checked and it seems we should upgrade the rds module to `4.0` because in `3.5` still this change is not applied.

Reply:
I just noticed we already have raised a ticket for this problem, so I revert this change as it will be addressed separately.